packages/jazz-tools/src/tests/groupsAndAccounts.test.ts

```diff
@@ -85,3 +86,95 @@ describe("Custom accounts and groups", async () => {
     expect(map._owner.castAs(CustomGroup).nMembers).toBe(2);
   });
 });
+
+describe("Group inheritance", () => {
+  class TestMap extends CoMap {
+    title = co.string;
+  }
+
+  test("Group inheritance", async () => {
+    const me = await Account.create({
+      creationProps: { name: "Hermes Puggington" },
+      crypto: Crypto,
+    });
+
+    const parentGroup = Group.create({ owner: me });
+    const group = Group.create({ owner: me });
+
+    group.extend(parentGroup);
+
+    console.log(
+      group.id,
+      group._raw.core.getDependedOnCoValuesUncached(),
+      parentGroup.id,
+    );
+
+    console.log(
+      (group._raw.core.getCurrentContent() as RawGroup)
+        .keys()
+        .filter((k) => k.startsWith("parent_"))
+        .map((k) => k.replace("parent_", "")),
+    );
```

Comment:
**question:** would it be ok to remove these console logs?

They make the test results harder to read.